addons/l10n_fi/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) Avoin.Systems 2016
+# Authors:
+#   Miku Laitinen (Avoin.Systems)
+#   Mikko Närjänen (Avoin.Systems)
+#   Toni Pesola (Avoin.Systems)
+
+# noinspection PyStatementEffect
+{
+    "name": "Finnish localization",
+    "version": "9.0.0.1.0",
+    "category": "Localization",
+    "description": """
+        General modifications useful to any Finnish company.
+        See README.md for details.
```

Comment:
Why not put everything in the description? The README file is not available to user that install a module, only to system administrators.